### DISCUSSION TITLE
[Feature] Album Description

### DISCUSSION THREAD

**Discussion by mirisbowring** [immich-app/immich#2490] [Feature Request]
### The feature

I would like to be able to set a description for an album.

Imagine you had a great week somewhere and would like to summarize some memories to keep them tight with the images.
I know that there is an description for images but i don't want to add the same description for every file.

Regards

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by jon-f-novabackup** [comment]
Great idea.

**Comment by jrasm91** [comment]
Implemented a few releases ago
